webapp/locales/de.json

```diff
@@ -351,8 +351,8 @@
     }
   },
   "followButton": {
-    "follow": "Folgen",
-    "following": "Folge Ich"
+    "follow": "Abonnieren",
+    "following": "Abonniere Mich"
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-21T11:05:46Z" title="Friday, February 21st 2020, 12:05:46 pm +01:00">Feb 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -351,8 +351,8 @@
     }
   },
   "followButton": {
-    "follow": "Folgen",
-    "following": "Folge Ich"
+    "follow": "Abonnieren",
+    "following": "Abonniere Mich"
```


This sounds strange. May bie just `Abonniert`?